### PR TITLE
 [Intl] Exclude root language

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | 4.4 
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no     
| Deprecations? | no
| Tests pass?   | yes    
| Fixed tickets | #...   
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Spotted in #32388

### CODE REVIEW

File: src/Symfony/Component/Intl/Data/Generator/LanguageDataGenerator.php [hunk 1]
```diff
@@ -84,6 +84,7 @@ class LanguageDataGenerator extends AbstractDataGenerator
         'zh' => 'zho',
     ];
     private static $blacklist = [
+        'root' => true,
```
Comment (javiereguiluz) [inline src/Symfony/Component/Intl/Data/Generator/LanguageDataGenerator.php]: Could we add a very brief comment explaining what's this ... like we do for the rest of special "languages"? Thanks.

File: src/Symfony/Component/Intl/Data/Generator/LanguageDataGenerator.php [hunk 2]
```diff
@@ -84,7 +84,7 @@ class LanguageDataGenerator extends AbstractDataGenerator
         'zh' => 'zho',
     ];
     private static $blacklist = [
-        'root' => true,
+        'root' => true, // Absolute root language
```
Comment (javiereguiluz) [inline src/Symfony/Component/Intl/Data/Generator/LanguageDataGenerator.php]: I'm sorry but this comment doesn't help much. What about something like: `Special non-existent language used for ...`

Comment (ro0NL) [reply src/Symfony/Component/Intl/Data/Generator/LanguageDataGenerator.php]: I dont know what it's used for :) `root` is not part of the ISO 639-3 codes.

AFAIK it's only used for grouping; if `en` is a root language, then `root` is the absolute language root.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ro0NL.

**Pull request merged**